pkg/apis/certificates/types.go

```diff
@@ -42,6 +42,19 @@ type CertificateSigningRequestSpec struct {
 	// Base64-encoded PKCS#10 CSR data
 	Request []byte
 
+	// Requested signer for the request. It is a qualified name in the form:
+	// `scope-hostname.io/name`.
+	// If empty, it will be defaulted for v1beta1:
```

Comment:
Can we only document this in v1beta1/types.go? Defaulting is specific to API versions.